policy/cnspec_policy.proto

```diff
@@ -966,10 +969,36 @@ message PurgeAssetsConfirmation {
   map<string, string> errors = 2;
 }
 
+// Sources is a list of sources.
+// Used to get past marshal/unmarshal limitations of repeated Source fields.
```

Comment:
Approaching this very naive; what limitation is that?

Reply:
You cannot call `proto.Marshal(...)` or unmarshal on a `repeated ...` field directly.